docs/resolving-link-and-linting-issues.md

```diff
@@ -0,0 +1,82 @@
+# Resolving Link and Linting Issues in Contributions
+
+To maintain the quality and reliability of our content, we've integrated linting
+and link validation checks into our GitHub Actions CI/CD pipeline. This document
+guides you through identifying issues flagged by these checks and resolving them
+using our `Makefile` commands.
+
+## Understanding the Checks
+
+Our CI/CD pipeline includes two primary checks for content contributions:
+
+1. **Markdown Linting**: Ensures that Markdown files adhere to our project's
+   formatting standards.
+2. **Link Validation**: Checks for broken links in the compiled site output to
+   ensure all internal and external links are valid.
+
+If either of these checks fails during a GitHub Actions run, your pull request
+will indicate a failure, requiring resolution before merging.
+
+## Resolving Issues
+
+### Markdown Linting Issues
+
+If the linting check fails, it typically means there are formatting
+discrepancies in your Markdown files.
+
+- **Identify Issues**: Review the output of the GitHub Actions run to see which
+  files and specific formatting rules were violated.
+- **Auto-Format Locally**: Run the following command in your local environment
+  to automatically format all Markdown files according to our project's
+  standards:
+
+  `bash make format-md`
+
+- **Review Changes**: Check the auto-formatted files to ensure changes are as
+  expected and commit them to your branch.
+
+### Link Validation Issues
+
+A failed link validation check indicates there are broken or invalid links in
+your content.
+
+- **Identify Broken Links**: Examine the GitHub Actions output to find which
+  links failed the validation check and in which files.
+- **Manual Correction**: Update your Markdown files to fix or replace the broken
+  links. Use valid URLs and ensure placeholders like `TBC` or `example.com` are
+  replaced with final links.
+- **Test Locally**: After making changes, you can build the site locally and run
```

Comment:
I'm not sure we can expect non-devs to be able to test locally. They might use the Github UI.

Reply:
Agreed. we are removing any lint checking from the GitHub Actions